### PR TITLE
9980: Reporting: Judge Activity Report - Progress Description

### PR DESCRIPTION
**Pull request by rgoshen**
This PR adds "Progress Description" to the Judge Activity Report.

--------------

Screenshot 2023-05-16 at 20 28 19

### CODE REVIEW

File: web-api/src/reports/getCasesByStatusAndByJudgeLambda.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { genericHandler } from '../genericHandler';
+
+/**
+ * retrieves closed cases associated with the specified judge
```
Comment (rosiehsmith) [inline web-api/src/reports/getCasesByStatusAndByJudgeLambda.ts]: Nit: docs

Comment (rgoshen) [reply web-api/src/reports/getCasesByStatusAndByJudgeLambda.ts]: Thanks...will update that

File: shared/src/persistence/elasticsearch/getDocketNumbersByStatusAndByJudge.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { search } from './searchClient';
+
+/**
+ * getCasesByStatusAndByJudge
```
Comment (rosiehsmith) [inline shared/src/persistence/elasticsearch/getDocketNumbersByStatusAndByJudge.ts]: Nit: docs

File: shared/src/proxies/reports/getCasesByStatusAndByJudgeProxy.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { post } from '../requests';
+
+/**
+ * getCasesClosedByJudgeInteractor
```
Comment (rosiehsmith) [inline shared/src/proxies/reports/getCasesByStatusAndByJudgeProxy.ts]: d-d-d-docs!

**Review by rosiehsmith — COMMENTED** [review]
Looks good! Great tests!

**Review by rosiehsmith — APPROVED** [review]

File: shared/src/persistence/elasticsearch/getDocketNumbersByStatusAndByJudge.ts (hunk 2 above)
Comment (rgoshen) [reply shared/src/persistence/elasticsearch/getDocketNumbersByStatusAndByJudge.ts]: updated...thx

File: shared/src/proxies/reports/getCasesByStatusAndByJudgeProxy.ts (hunk 3 above)
Comment (rgoshen) [reply shared/src/proxies/reports/getCasesByStatusAndByJudgeProxy.ts]: thx again...updated

**Pull request merged**